examples/workflow/2_generate_data.ipynb

```diff
@@ -0,0 +1,209 @@
+    "Now all building blocks for generating our data sample are at hand and you can define a data sample of arbitrary size!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pycompwa.ui as pwa  # interface to ComPWA"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "particle_list = pwa.read_particles('model.xml')\n",
+    "\n",
+    "kinematics = pwa.create_helicity_kinematics('model.xml', particle_list)\n",
+    "\n",
+    "generator = pwa.EvtGenGenerator(\n",
+    "    kinematics.get_particle_state_transition_kinematics_info())\n",
+    "\n",
+    "random_generator = pwa.StdUniformRealGenerator(12345)\n",
+    "\n",
+    "phsp_sample = pwa.generate_phsp(int(1e5), generator, random_generator)\n",
+    "\n",
+    "intensity_builder = pwa.IntensityBuilderXML(\n",
+    "    'model.xml', particle_list, kinematics, phsp_sample)\n",
+    "\n",
+    "intensity = intensity_builder.create_intensity()\n",
+    "\n",
+    "data_sample = pwa.generate(\n",
+    "    int(1e4), kinematics, generator, intensity, random_generator)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note::\n",
+    "   :mod:`pycompwa.ui` is the python interface to\n",
+    "   `ComPWA's C++ modules <https://compwa.github.io/ComPWA/annotated.html>`__.\n",
+    "   It has four major important components for evaluating intensities:\n",
+    "  \n",
+    "   * The :class:`~.EventCollection` class is simply an event-based list of\n",
+    "     four-momenta with a list of Particle IDs.\n",
+    "   \n",
+    "   * The :class:`~.Kinematics` class allows one to compute kinematic\n",
+    "     variables, like :math:`(s,\\theta,\\phi)` in the helicity formalism,\n",
+    "     from the momentum tuples in an :class:`.EventCollection`.\n",
+    "   \n",
+    "   * The :class:`~.DataSet` class is the result when you\n",
+    "     :meth:`~pycompwa.ui.Kinematics.convert` an\n",
+    "     :class:`~.EventCollection`. It is essentially a table of kinematic\n",
+    "     variables for each event in that original collection of four-momenta.\n",
+    "\n",
+    "   * The :class:`~.Intensity` class allows one to compute an intensity (an\n",
+    "     absolute value) for each data point in the :class:`~.DataSet`. You do\n",
+    "     this computation with the\n",
+    "     :meth:`~pycompwa.ui.FunctionTreeIntensity.evaluate` method (see\n",
+    "     :doc:`4_analyze_results`)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.2 Exporting and importing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ``pycompwa.ui`` module allows you to export the data samples to two different file formats: [ROOT files](https://root.cern.ch/root-files) and ASCII files."
```

Comment:
Link to the module

Reply:
https://github.com/ComPWA/pycompwa/pull/74#discussion_r400882556